cnxarchive/tests/data/README.rst

```diff
@@ -1,20 +1,28 @@
 :data.sql:
     Generated via::
         cnx-archive-initdb cnxarchive/tests/testing.ini --with-example-data
-        # Initialize the schema manager
-        dbmigrator --config cnxarchive/tests/testing.ini init
-        # Create the schema migration step
-        dbmigrator generate <description of the change>
-        # Make needed changes via db-migrator migrations
-        dbmigrator --config cnxarchive/tests/testing.ini migrate
-        pg_dump -a -T licenses -T tags -T roles --disable-triggers --inserts cnxarchive-testing > data.sql
+        pg_dump -a -T schema_migrations -T licenses -T tags -T roles -T modulestates -T service_states --disable-triggers --inserts cnxarchive-testing > cnxarchive/tests/data/data.sql
 
-
-
-:legacy-data.sql: Contains a data set extracted from a legacy database
-    structure that was populated using cnx-populate. This
-    data is a SQL dump using::
-
-        pg_dump --data-only -T licenses -T tags -T trees \
-          --inserts --disable-triggers \
-          -d $DB_NAME > legacy-data.sql
+        If you have a a feature branch with schema and/or data changes, you'll
+        need to build a migration. Which is basically a schema-diff.
+            git checkout myfeature
+            cnx-archive-initdb cnxarchive/tests/testing.ini
+            # dump a schema
+            dbtoyaml cnxarchive-testing -o cnxarchive-myfeature.yml
+            dropdb cnxarchive-testing; createdb -O cnxarchive cnxarchive-testing
+            git checkout master
+            cnx-archive-initdb cnxarchive/tests/testing.ini
+            # dump a schema
+            dbtoyaml cnxarchive-testing -o cnxarchive-master.yml
```

Comment:
yeah, thought about that. The existing steps where unclear, so I documented what I did for this case. :-)